src/lib/mdns/Advertiser_ImplMinimalMdns.cpp

```diff
@@ -596,15 +637,87 @@ FullQName AdvertiserMinMdns::GetCommisioningTextEntries(const CommissionAdvertis
         sprintf(txtVidPid, "VP=%d", params.GetVendorId().Value());
     }
 
-    char txtPairingInstrHint[128];
-    if (params.GetPairingInstr().HasValue() && params.GetPairingHint().HasValue())
+    char txtDeviceType[32];
+    if (params.GetDeviceType().HasValue())
     {
-        sprintf(txtPairingInstrHint, "P=%s+%d", params.GetPairingInstr().Value(), params.GetPairingHint().Value());
-        return AllocateQName(txtDiscriminator, txtVidPid, txtPairingInstrHint);
+        sprintf(txtDeviceType, "DT=%d", params.GetDeviceType().Value());
+    }
+    else
+    {
+        sprintf(txtDeviceType, "");
```

Comment:
strcpy here an below. I think our CI compilers already complain about this.

How about not including this element at all if missing. I am not sure our libraries like having empty strings in there (at least the minmdns implementation)